packages/mui-material/src/Tabs/Tabs.test.js

```diff
@@ -1438,5 +1441,93 @@ describe('<Tabs />', () => {
       expect(hasLeftScrollButton(container)).to.equal(false);
       expect(hasRightScrollButton(container)).to.equal(false);
     });
+
+    // https://github.com/mui/material-ui/issues/41388
+    it('should allow a user to click a scrollable Tab without an error', async function test() {
+      function DynamicScrollableTabs() {
```

Comment:
Analogous to [DynamicScrollableTabs](https://github.com/nicholeuf/mui-test-fixture-41388/blob/main/app/components/DynamicScrollableTabs.tsx) used in the live example.